content/manager_webui/maintenance.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bt_wiki
+title: Maintenance Mode
+category: Manager
+draft: false
+abstract: Cloudify's Web Interface
+weight: 200
+---
+
+Managing maintenance mode page will be found at the settings section in the Web UI. Upon routing you will be presented with the manager maintenance mode status and a button used to change it.<br/>
```

Comment:
replace with `The Maintenance Mode page is found in the settings section, where you may see and toggle the manager's maintenance mode.`